### DISCUSSION TITLE
How to get new auth data when updated user_meta?

### DISCUSSION THREAD

**Discussion by wiwatsrt** [supabase/supabase#902] [Questions]
I updated user meta with `auth.update()`
And check with `auth.onAuthStateChange()` when triggers with event  USER_UPDATED
Why session.user same old data, Need reSignIn only for change data?

**Comment by ydennisy** [comment]
@wiwatsrt did you ever get an answer to this?

We are having the same issue with the google auth sign in flow.

@kiwicopple could you please take a look?
